src/main/scala/com/microsoft/hyperspace/index/IndexManager.scala

```diff
@@ -54,7 +54,9 @@ trait IndexManager {
   def restore(indexName: String): Unit
 
   /**
-   * Does hard delete of indexes marked as `DELETED`. Once vacuumed, an index can't be 'restore'd.
+   * Does hard delete of indexes if it is marked as `DELETED`.
+   * Does clean up of indexes (hard delete of unused indexes) if it is 'ACTIVE'.
```

Comment:
Could you revise the comment? it would be good these two lines in one sentence.

Reply:
How about the new one?